src/test/java/it/gov/pagopa/payhub/activities/util/FilesUtils.java

```diff
@@ -0,0 +1,15 @@
+package it.gov.pagopa.payhub.activities.util;
+
+import java.io.IOException;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+public class FilesUtils {
```

Comment:
is this method really generic? it seems to me to be a method which will not used outside the context of a zip file... let's move this method on the class itself, removing this class